package.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>sdformat_urdf</name>
+  <version>0.1.0</version>
+  <description>
+    URDF plugin to parse SDFormat XML into URDF C++ structures.
```

Comment:
Is it fair to restate `URDF C++ structures` as `URDF C++ DOM objects`?

Reply:
`structures` -> `DOM objects` in aff9def